specifications/api/swagger.yaml

```diff
@@ -752,6 +761,32 @@ components:
         outputs: *transactionOutputs
 
     ApiStakePool: &ApiStakePool
+      type: object
+      required:
+        - id
+        - relative_stake
+        - non_myopic_member_rewards
+        - produced_blocks
+        - apparent_performance
+        - cost
+        - margin
+        - saturation
+      properties:
+        id: *stakePoolId
+        relative_stake:
+          <<: *percentage
+          description: The pools stake compared to the stake of all pools.
+        cost: *stakePoolCost
+        margin: *stakePoolMargin
+        saturation: *stakePoolSaturation
+        non_myopic_member_rewards: *nonMyopicMemberRewards
+        metadata: *stakePoolMetadata
+        produced_blocks:
+          <<: *numberOfBlocks
+          description: Number of blocks produced by a given stake pool.
+        apparent_performance: *stakePoolApparentPerformance
```

Comment:
Do we actually still have this one on cardano-node ?

Reply:
Are we going to display some form of `produced_blocks`? Then why not? Though it will be a different apparent performance from that used to calculate the non-myopic rewards.

Reply:
We also need the pool's stake on the previous epoch, do we have that?

Reply:
I'm not sure. If LSQ worked for k blocks old, then we have stake distributions for the k latest blocks.

Should we remove?

Should we not also remove `produced_blocks`? (Or have say `lifetime_blocks` if we keep it?)

Reply:
Ideally we would have both last production and total production. Though we do not depend on the LSQ for these, we have this information from the chain sync data. 

> I'm not sure. If LSQ worked for k blocks old, then we have stake distributions for the k latest blocks. Should we remove?

Well, without the stake distribution of past epochs, we are running into the same problem / awkwardness as during the ITN where we would have holes in our dataset. I'd rather not go through this again.

Reply:
> Well, without the stake distribution of past epochs, we are running into the same problem / awkwardness as during the ITN where we would have holes in our dataset. I'd rather not go through this again.

To be clear: We are _guaranteed_ to have `k` blocks with corresponding stake distributions. On the ITN we weren't guaranteed to have that.

Reply:
What would the point of  `last_production` be, and what would the window be? Wouldn't it just be to give users some notion of the apparent performance?

Then why not either remove _both_ `last_production` and `apparent_performance`, or keep both? 🤔